src/wasm-lib/kcl/src/unparser.rs

```diff
@@ -280,6 +281,44 @@ impl ArrayExpression {
     }
 }
 
+/// An expression is syntactically trivial: i.e., a literal, identifier, or similar.
+fn expr_is_trivial(expr: &Expr) -> bool {
```

Comment:
Perhaps should be a method on `Expr`?

Reply:
I thought about that, my feeling was that it was specific enough to the context here that it shouldn't be on expr (I imagine a more general `is_trivial` method might be a bit different to this one (specifically for formatting within a range)